apps/frontend/src/pages/quiz/libs/components/quiz-form/quiz-form.tsx

```diff
@@ -0,0 +1,20 @@
+import { QuizQuestion } from "~/libs/components/components.js";
+import { useAppForm } from "~/libs/hooks/hooks.js";
+
+type FormValues = {
+	value: string;
+};
```

Comment:
This seems like incorrect type, we will have multiple questions, so we should be able to save multiple answers

Reply:
I thought the form would be in another task, so I added this just to add my quiz question component.

Reply:
Understood, let's keep in that case